examples/typescript-nodejs/README.md

```diff
@@ -0,0 +1,45 @@
+Example Node.js project consuming TypeScript types
+==================================================
+
+This Node.js project consumes the [Digital Identity Vocab](https://github.com/alphagov/di-identity-vocab) TypeScript types.
+
+In the `package.json`, note the `@alphagov/di-identity-vocab` dependency:
+
+```json
+"@alphagov/di-identity-vocab": "1.3.0"
+```
+
+> **Note**
+> See [releases](https://github.com/alphagov/di-identity-vocab/releases) for the latest version.
+
+### Build the project
+
+##### Prerequisities
+
+This project uses TypeScript and expects the `tsc` command to be available.
+
+Ensure you have authenticated to the `@alphagov` npm scope in GitHub Packages:
+
+```shell
+npm login --scope=@alphagov --auth-type=legacy --registry=https://npm.pkg.github.com
```

Comment:
Is this still `@alphagov` or `gov.uk-one-login` for those who are not familiar using github repos for `npm` I think pre-requisite for this to work should be mentioned. PAT and the scopes required would be helpful

Reply:
Thank you for catching this - updated to the new org name 👍